libfwupd/fwupd-self-test.c

```diff
@@ -533,6 +535,13 @@ fwupd_common_machine_hash_func (void)
 		g_test_skip ("Missing /etc/machine-id");
 		return;
 	}
+	if (!g_file_get_contents ("/etc/machine-id", &buf, &sz, &error))
+		return;
```

Comment:
I think you want a g_test_skip here if the /etc/machine-id is unreadable -- and with that LGTM. Thanks.